### PR TITLE
Add some basic template nav2 headers, we can improve them later

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers

### CODE REVIEW

File: shared/wallets/routes.js [hunk 1]
```diff
@@ -167,19 +167,21 @@ class WalletsSubNav extends React.PureComponent<any> {
   }
 }
 
+const WalletsSubNavigator = createNavigator(
```
Comment (buoyad) [inline shared/wallets/routes.js]: Mobile gets mad about this
image

**Comment by cjb** [comment]
@buoyad Oops, thanks! Maybe this, then?

File: shared/wallets/routes.js [hunk 2]
```diff
@@ -167,21 +167,28 @@ class WalletsSubNav extends React.PureComponent<any> {
...
-}
-
 export const newRoutes = {
   'tabs.walletsTab': {
-    getScreen: () => isMobile ? require('./wallet/container').default : WalletsSubNavigator, upgraded: true,
+    getScreen: () => {
+      if (isMobile) {
+        return require('./wallet/container').default
+      } else {
+        const WalletsSubNavigator = createNavigator(
```
Comment (buoyad) [inline shared/wallets/routes.js]: Creating a new navigator in here strikes me as fishy - how often does react navigation call this?

Comment (cjb) [reply shared/wallets/routes.js]: Ah, looks like only once per launch: added a console.warn() and it only fired once.

**Pull request merged**